### PR TITLE
Patternlab/DP-16073:  Fine tune campaign page styling

### PR DESCRIPTION
**Pull request by avrilpearl**
___DESCRIPTION___
Fixed
- (Patternlab) [MarketingCampaign] DP-16073: Fix spacing between marketing campaign page sections. #772
- (Patternlab) [Card] DP-16073: Stack image on primary card on tablet size screen. #772
- (Patternlab) [CampaignVideo] DP-16073: Remove heading style override and wrapper top/bottom padding in preference of inheriting margins from page. #772

Testing:
1. Go to http://localhost:3000/?p=pages-marketing-campaign
2. Confirm header/spacing between sections is appropriate / default mayflower
2. Confirm primary card image/content stacks at medium breakpoint (780px) vs small as before

### CODE REVIEW

**Review by ygannett — APPROVED** [review]
Looks good.

File: assets/scss/04-templates/_marketing-campaign.scss [hunk 1]
```diff
@@ -1,3 +1,13 @@
 .ma__marketing-campaign__main-content {
-  margin-bottom: 0px;
+  .page-content>:last-child {
+    &.ma__key-message {
+      margin-bottom: -45px;
+      &--c-white {
+        margin-top: 0px;
```
Comment (clairesunstudio) [inline assets/scss/04-templates/_marketing-campaign.scss:6]: this works but I would consider removing the key message padding instead

**Comment by clairesunstudio** [comment]
In MarketingCampaign page, can you use key message for the first paragraph instead of rich text directly so that it's center aligned?

**Review by clairesunstudio — APPROVED** [review]
looks good in mayflower!

**Pull request merged**